portfolio/src/main/webapp/index.html

```diff
@@ -347,6 +341,23 @@ <h2><span class="underline-magical food-diary">food diary</span></h2>
                 </div>
             </div>
 
+             <!-- Comments Form -->
+             <div class= "row">
+                <a id="comments-section"></a>
+                <h2><span class="underline-magical food-diary">comments</span></h2>
+                <p style="font-style: italic; margin-bottom: 20px">please fill out the form to leave a comment</p>
```

Comment:
I generally recommend against inline styles since those styles have a higher specificity and overwrite any other styles that might be applied to that same element. Here's a helpful guide that explains CSS specificity and how conflicting CSS styles on the same element are applied: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

Reply:
made this into a class